QuizApp/QuizSession/ViewModel/QuizSessionViewModel.swift

```diff
@@ -0,0 +1,38 @@
+import Combine
+import UIKit
+
+class QuizSessionViewModel {
```

Comment:
Instead of storing currentQuestionIndex, viewModel could publish a current question model. 
That way when you call viewModel.nextQuestion() new question is published and viewcontroller would be subscribed to it.